ImageGallery/src/org/sleuthkit/autopsy/imagegallery/ImageGalleryModule.java

```diff
@@ -53,262 +52,267 @@
         return Paths.get(theCase.getModuleDirectory(), getModuleName());
     }
 
-    /** provides static utilities, can not be instantiated */
+    /**
+     * Prevents instantiation.
+     */
     private ImageGalleryModule() {
     }
 
-    /** is listening enabled for the given case
+    /**
+     * Indicates whether or not the image gallery module is enabled for a given
+     * case.
      *
-     * @param c
+     * @param theCase The case.
      *
-     * @return true if listening is enabled for the given case, false otherwise
+     * @return True or false.
      */
-    static boolean isEnabledforCase(Case c) {
-        if (c != null) {
-            String enabledforCaseProp = new PerCaseProperties(c).getConfigSetting(ImageGalleryModule.MODULE_NAME, PerCaseProperties.ENABLED);
-            return isNotBlank(enabledforCaseProp) ? Boolean.valueOf(enabledforCaseProp) : ImageGalleryPreferences.isEnabledByDefault();
-        } else {
-            return false;
-        }
+    static boolean isEnabledforCase(Case theCase) {
+        String enabledforCaseProp = new PerCaseProperties(theCase).getConfigSetting(ImageGalleryModule.MODULE_NAME, PerCaseProperties.ENABLED);
+        return isNotBlank(enabledforCaseProp) ? Boolean.valueOf(enabledforCaseProp) : ImageGalleryPreferences.isEnabledByDefault();
     }
 
     /**
-     * Is the given file 'supported' and not 'known'(nsrl hash hit). If so we
-     * should include it in {@link DrawableDB} and UI
+     * Indicates whether or not a given file is of interest to the image gallery
+     * module (is "drawable") and is not marked as a "known" file (e.g., is not
+     * a file in the NSRL hash set).
      *
-     * @param abstractFile
+     * @param file The file.
      *
-     * @return true if the given {@link AbstractFile} is "drawable" and not
-     *         'known', else false
+     * @return True if the file is "drawable" and not "known", false otherwise.
      *
-     * @throws
-     * org.sleuthkit.autopsy.modules.filetypeid.FileTypeDetector.FileTypeDetectorInitException
+     * @throws FileTypeDetectorInitException If there is an error determining
+     *                                       the type of the file.
      */
-    public static boolean isDrawableAndNotKnown(AbstractFile abstractFile) throws FileTypeDetector.FileTypeDetectorInitException {
+    private static boolean isDrawableAndNotKnown(AbstractFile abstractFile) throws FileTypeDetector.FileTypeDetectorInitException {
         return (abstractFile.getKnown() != TskData.FileKnown.KNOWN) && FileTypeUtils.isDrawable(abstractFile);
     }
 
     /**
-     * Listener for IngestModuleEvents
+     * A listener for ingest module application events.
      */
     static private class IngestModuleEventListener implements PropertyChangeListener {
 
         @Override
-        public void propertyChange(PropertyChangeEvent evt) {
-            if (RuntimeProperties.runningWithGUI() == false) {
```

Comment:
I believed that the check was a bug, since one of the effects of handling a FILE_DONE event is an update to the groups table in the case database. It sounds like it might not matter after all. I will add a TODO to the code and circle back to this issue. As it is, there is no harm in doing some potential extra work on an auto ingest node.